### ISSUE TITLE
[IDEA/REQUEST] change background color

### ISSUE THREAD

**Issue by imeed166** [jdanielmourao/obsidian-sanctum#91]
**Is your feature request related to a problem? Please describe.**
I would like to change the default background color.

**Describe the solution you'd like**
slider for hue, brightness and saturation like shimmerring focus

**Additional context**
![image](https://user-images.githubusercontent.com/38974541/147887921-34e1bf00-3fcc-4351-aef5-eb4aa1e437ab.png)

**Comment by jdanielmourao** [comment]
Hi!
The thing about having HSL sliders is that HSL doesn't abide by the colour models I used to make all the theme's colours. Basically, if you changed Brightness, for example, it would output different levels of brightness from the ones I originally set. This means that some shades would lose their contrast ratio, and would ruin the theme from an accessibility perspective.
If you use HSL to make 2 colours, both with the same HSL Brightness level, those 2 colours would in fact have different perceived brightness levels, since the human eye perceives each colour differently. The colour model I used takes that into account.
I've thought about including a hue slider, for example, but I could also make a group of pre-set colour palettes of which I have higher control of (some of the colours are already shown in the headings).
